README.md

```diff
@@ -212,7 +212,7 @@ Allows access to the file system.
 
 ---
 
-## [SMS Receiver API](https://web.dev/sms-receiver-api-announcement/)
+## [WebOTP API](https://web.dev/sms-receiver-api-announcement/)
```

Comment:
It's now at https://web.dev/web-otp/ it seems.

Reply:
it redirects

Reply:
I'm still seeing a 404 at https://web.dev/sms-receiver-api-announcement/

Reply:
Ah yes, after a hard reload, I see it now.